Libraries/Image/ImageStylePropTypes.js

```diff
@@ -34,6 +34,15 @@ var ImageStylePropTypes = {
   // It changes the color of all the non-transparent pixels to the tintColor
   tintColor: ColorPropType,
   opacity: ReactPropTypes.number,
+  
+  /**
+  * Android-Specific style to allow rounded corners based on a solid color.
+  * Specifying an overlayColor will cause Fresco to switch to its other rounding 
+  * corners rendering mode, OVERLAY_COLOR, and will draw rounded corners (configured using borderRadius) 
+  * by overlaying the solid color specified.
+  * a null or transparent overlayColor will default back to BITMAP_ONLY.
```

Comment:
Please add a line with `@platform android` at the end. This has effect on how the docs render on the website (grouped by platform). See e.g.
https://github.com/facebook/react-native/blob/master/Libraries/Components/ScrollView/ScrollView.js#L178
https://facebook.github.io/react-native/docs/scrollview.html#content